zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AdminResponse.java

```diff
@@ -37,4 +38,8 @@ public int getLength() {
   public void write(final MutableDirectBuffer buffer, final int offset) {
     bodyEncoder.wrapAndApplyHeader(buffer, offset, headerEncoder);
   }
+
+  public String getConfiguration() {
+    return configuration;
```

Comment:
:wrench: Instead of a `configuration` field, just return `bodyDecoder.payload()` here.